src/main/java/net/minecraftforge/fml/hooks/BasicEventHooks.java

```diff
@@ -119,4 +120,14 @@ public static void onPostServerTick()
     {
         MinecraftForge.EVENT_BUS.post(new TickEvent.ServerTickEvent(TickEvent.Phase.END));
     }
+
+    public static void onPreEnvironmentTick(World world, Chunk chunk, int randomTickSpeed)
+    {
+        MinecraftForge.EVENT_BUS.post(new TickEvent.WorldTickEvent.EnvironmentTickEvent(LogicalSide.SERVER, TickEvent.Phase.START, world, chunk, randomTickSpeed));
+    }
+    
+    public static void onPostEnvironmentTick(World world, Chunk chunk, int randomTickSpeed)
+    {
+        MinecraftForge.EVENT_BUS.post(new TickEvent.WorldTickEvent.EnvironmentTickEvent(LogicalSide.SERVER, TickEvent.Phase.END, world, chunk, randomTickSpeed));
```

Comment:
```java
new TickEvent.WorldTickEvent.EnvironmentTickEvent
```
looks wrong here, seeing that `EnvironmentTickEvent` now directly extends `TickEvent`. It should just be `new TickEvent.EnvironmentTickEvent()` for simplicity, even if this does technically compile just fine.